manifests/init.pp

```diff
@@ -204,6 +210,8 @@
   Integer[0] $content_service_worker_timeout = 90,
   Integer[0] $api_service_worker_timeout = 90,
   Hash[String[1], String[1]] $api_client_auth_cn_map = {},
+  Boolean $cache_enabled = false,
+  Optional[Variant[Integer[0], Enum['None']]] $cache_expires_ttl = undef,
```

Comment:
What does it mean to have `CACHE_SETTINGS = { 'EXPIRES_TTL': 0 }` and is that a valid configuration? Is it meaningfully different from `CACHE_SETTINGS = { 'EXPIRES_TTL': None }` ?

The docs string calls out `None` as an acceptable value but doesn't explain what happens if this is `0` instead; therefore should we use `Integer[1]` here?

Reply:
Based on https://github.com/pulp/pulpcore/commit/81684b206e3fd8bceb104e4a77c0d717387667f9#diff-a8f18c64735759c7f007d734b8b40e45938d03bc54327381fa2c9cf5fa2df544R42-R43 ,

`0` should have the exact same behavior as `None`

Reply:
That is a good point. I was aiming for following their docs rather than the code: https://github.com/pulp/pulpcore/commit/81684b206e3fd8bceb104e4a77c0d717387667f9#diff-9243f057fa8b6379f8ecb2570d457b290cf2e455e6d591541022caefc0b81448R215

We should probably point this out to them. As I would interpret 0 as meaning effectively this never gets cached.

Reply:
Big :+1: for expanding upstream documentation. That should really be specified. If we can stick to just integers then that would be my preference.

Reply:
You are right,`0` and `None` have the exact same behavior currently, I'll add the clarification to the docs.